Formula/falcon.rb

```diff
@@ -1,7 +1,8 @@
 class Falcon < Formula
   desc "Multi-paradigm programming language and scripting engine"
   homepage "http://www.falconpl.org/"
-  url "http://falconpl.org/project_dl/_official_rel/Falcon-0.9.6.8.tgz"
+  url "https://www.mirrorservice.org/sites/distfiles.macports.org/falcon/Falcon-0.9.6.8.tgz"
+  mirror "http://pkgs.fedoraproject.org/repo/pkgs/Falcon/Falcon-0.9.6.8.tgz/8435f6f2fe95097ac2fbe000da97c242/Falcon-0.9.6.8.tgz"
   sha256 "f4b00983e7f91a806675d906afd2d51dcee048f12ad3af4b1dadd92059fa44b9"
 
   head "http://git.falconpl.org/falcon.git"
```

Comment:
Looks like the head URL is down, but I found https://github.com/falconpl/falcon.git as an alternate?

Reply:
The website says that's just a mirror but https://github.com/falconpl/falcon/issues/4